### PR TITLE
add credentials support for DomainTools Iris integration

### PR DESCRIPTION
**Pull request by pdragoi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
 urllib3.disable_warnings()
 
 ''' GLOBALS '''
-USERNAME = demisto.params().get('username')
-API_KEY = demisto.params().get('apikey')
+USERNAME = demisto.params().get('credentials', {}).get('identifier')
```
Comment (mmhw) [inline Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py:19]:
```suggestion
USERNAME = demisto.params().get('credentials', {}).get('identifier') or demisto.params().get('username')
```

File: Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py [hunk 2]
```diff
@@ -16,8 +16,8 @@
...
+API_KEY = demisto.params().get('credentials', {}).get('password')
```
Comment (mmhw) [inline Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py:20]:
```suggestion
API_KEY = demisto.params().get('credentials', {}).get('password') or demisto.params().get('apikey')
```

File: Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.yml [hunk 3]
```diff
@@ -4,13 +4,10 @@ commonfields:
   version: -1
 configuration:
 - display: API Username
```
Comment (mmhw) [inline Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.yml:6]: You should leave the parameters `username` and `apikey`, and change to `required: false` and add `hidden: true`, and add a new parameter as follows:
```
- display: API Username
  name: credentials
  type: 9
  required: false
  displaypassword: API Key
```

**Review by mmhw — COMMENTED** [review]
Hi @pdragoi ,
Thank you for your contribution!
Good work :)

Please note my comments below, as the aim is to maintain backwards compatibility.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by pdragoi** [comment]
> Hi @pdragoi , Thank you for your contribution! Good work :)
> 
> Please note my comments below, as the aim is to maintain backwards compatibility.
> 
> Please feel free to reach out to me with any questions - I'm available here or on slack :) Thanks again

I have implemented the changes. Please let me know if anything else is needed. Thank you!

**Review by mmhw — APPROVED** [review]

**Pull request merged**